docs/install.md

```diff
@@ -5,38 +5,28 @@ post installation to ensure the operator can function correctly.
 
 ## STS Clusters
 
-### Post operator installation
+### Pre operator installation
 
 In an STS Cluster, `CredentialsRequest`s are not automatically provisioned by
-the **cloud-credential-operator** and the manual intervention done by the
-cluster-admin is required. IAM role and policies as well as the credentials secret need to be provisioned manually for the further consumption by the pods.
+the **cloud-credential-operator** and manual intervention is required by the
+cluster-admin. IAM roles and policies as well as the credentials secret need to be provisioned manually for the further consumption by the pods.
 `ccoctl` binary can be used to facilitate this task.
 
 Normally, the **aws-load-balancer-operator** relies on the **cloud-credential-operator**
-to provision the secret for the operated controller using `CredentialsRequest`. And so in an STS cluster this
-secret needs to be provisioned manually. The **aws-load-balancer-operator** will wait until the required
-secret is created and available before spawning the **aws-load-balancer-controller** pod.
+to provision the secret for both the operator and provision controller using `CredentialsRequest`. And so in an STS cluster this
+secret needs to be provisioned manually. The **aws-load-balancer-operator** will fail to create pods if the 
```

Comment:
`to wait` was the right wording. [The operator will re enqueue](https://github.com/openshift/aws-load-balancer-operator/blob/ee2bba8824fde90abefb55fd43b70fbea22c07f0/pkg/controllers/awsloadbalancercontroller/controller.go#L158-L163) until the secret is created.